documentation/book/security.adoc

```diff
@@ -31,31 +41,31 @@ NOTE: "certificates rotation" for generating new ones on their expiration will b
 
 The data encryption is provided on two different listeners exposed by each Kafka broker.
 
-Interbroker communication is through the `REPLICATION` listener on port 9091, which is encrypted by default.
+Communication between brokers and between the different Kafka and Strimzi components is done through the `REPLICATION` listener on port 9091, which is encrypted by default.
 
 Encrypted communication with clients is provided through the `CLIENTTLS` listener on port 9093.
 
 NOTE: Un-encrypted communication with clients is still possible through the `CLIENT` listener on port 9092.
 
 === Clients connection via TLS
 
-If a Kafka client wants to connect to the encrypted listener (CLIENTTLS) on port 9093, it needs to trust the client's CA certificate in order to verify the broker certificate received during the SSL/TLS handshake.
-The clients CA certificate can be extracted from the generated `<cluster-name>-kafka-cert` Secret with the following command if the Kafka cluster is running on {OpenShiftName}.
+If a Kafka client wants to connect to the encrypted listener (CLIENTTLS) on port 9093, it needs to trust the cluster CA certificate in order to verify the broker certificate received during the SSL/TLS handshake.
+The cluster CA certificate can be extracted from the generated `<cluster-name>-cluster-ca-cert` Secret with the following command if the Kafka cluster is running on {OpenShiftName}.
 
 [source,shell]
-oc get secret <cluster-name>-kafka-cert -o jsonpath='{.data.clients-ca\.crt}' | base64 -d > clients-ca.crt
+oc get secret <cluster-name>-cluster-ca-cert -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
 
 ifdef::Kubernetes[]
 If the Kafka cluster is running on {KubernetesName}, the same result can be achieved with the following command.
 
 [source,shell]
-kubectl get secret <cluster-name>-kafka-cert -o jsonpath='{.data.clients-ca\.crt}' | base64 -d > clients-ca.crt
+kubectl get secret <cluster-name>-cluster-ca-cert -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
 
 endif::Kubernetes[]
 The native Kafka client, in order to use it, needs the certificate to be imported in a truststore using the `keytool` command line tool.
```

Comment:
This reads a little strangely. "The Kafka client will need to be configured to trust certificates signed by this CA. This is done by importing the CA certificate into the JVM's truststore using the `keytool` command, as follows:"